maven-plugin-adventure/README.md

```diff
@@ -1,45 +1,60 @@
 # Maven Plugin Adventure
 
-In this adventure we are going to see how to use open rewrite in a Maven project and run a recipe to migrate to Spring Boot 3. OpenRewrite recipes are lego blocks and therefore this process consists of multiple steps that are invisible for you:
+In this adventure, you will migrate an old version of the
```

Comment:
Now, looking at it we should make explicit that we will use the rewrite maven plugin.